packages/electron-app/src/main/index.ts

```diff
@@ -49,6 +89,17 @@ app.on('activate', function () {
   }
 });
 
+app.on('web-contents-created', (event, contents) => {
+  contents.on('will-attach-webview', (event, webPreferences, params) => {
```

Comment:
Since we don't have any webviews, this event will never be called afaict. Do we still need it?

And if we do decide to put it, I guess a `event.preventDefault()` inside should be enough?

Reply:
My thinking when I included it just as a have and don't need it rather than the other way around.